lib/chef/audit/runner.rb

```diff
@@ -79,11 +79,15 @@ def require_deps
         require 'rspec'
         require 'rspec/its'
         require 'specinfra'
+        require 'specinfra/helper'
+        require 'specinfra/helper/set'
         require 'serverspec/helper'
         require 'serverspec/matcher'
         require 'serverspec/subject'
         require 'chef/audit/audit_event_proxy'
         require 'chef/audit/rspec_formatter'
+
+        Specinfra::Backend::Cmd.send(:include, Specinfra::Helper::Set)
```

Comment:
Yeah, seems like we should follow up with them, I guess it works if you use their spec_helper but that seems suboptimal. If this is what we need to do to ship this in the short term, it's cool, but we should follow up on a "less scary" solution for the long term.